cmd/mms/main.go

```diff
@@ -33,25 +33,31 @@ func main() {
 	listFlags := []cli.Flag{
 		&cli.StringFlag{
 			Name:  "production-hub", // HTTP
-			Usage: "The production hub HTTP URL",
+			Usage: "The production hub HTTP URL.",
```

Comment:
This can be both, HTTP and HTTPS, what if we drop HTTP here?

Reply:
Technically, this and the comment below have nothing to do with this PR, but are related to the HTTPS PR and should be resolved separately.

/end nitpick